docs/xenserver-xencert-guide.md

```diff
@@ -52,7 +52,7 @@ Published Oct 2023
 
 ### Overview  
 
-The purpose of this document is to familiarize the reader with XenServer Shared Storage Certification Kit. The certification kit is designed to certify the interoperability of various types of storage hardware with XenServer.  
+The purpose of this document is to familiarize the reader with XenServer Shared Storage Certification Kit. The certification kit is designed to certify the interoperability of various types of storage hardware with XenServer. XenServer certification kit requires run with XenServer latest veriosn, ensure that XenServer is updated to the latest version before test.
 

```

Comment:
Hi, 
"requires run with XenServer latest version" - this is not correct in general.
At the moment, each XS certification kit release runs against its mapped XenServer release, e.g. 8.2 CU1 cert kits run to the latest of 8.2 CU1, and XS 8 cert kits run with the latest XS 8 release.
We can release XS cert kits with backward comp-abilities (i.e. release cert kits as separated deliverable and make it run with all XS releases), however, that needs to be verified with all released XS releases, and this also needs to change the HCL website's release page (there has per release based Cert Kits at the moment)
 #

Reply:
Agree, this is specific for XenServer 8 release

Reply:
"Make sure that XenServer8 has been updated to the latest version before testing. " this seems not necessary if we already said latest XenServer 8 release previously.